src/main/kotlin/calculator/domain/Separator.kt

```diff
@@ -3,4 +3,16 @@ package calculator.domain
 @JvmInline
 value class Separator(
     val separator: String
-)
+) {
+    companion object {
+        private const val REGEX_PATTERN = "//(.)\n(.*)"
+
+        fun create(text: String): Separator {
+            val regexResult = Regex(REGEX_PATTERN).find(text)
+            val customSeparator = regexResult?.let {
+                it.groupValues[1]
+            }
+            return Separator(customSeparator.toString())
```

Comment:
```suggestion
            return Separator(customSeparator.toString())
```

`toString()`으로 한다면 customSeparator가 null인 경우에 "null"을 반환하겠죠? 코틀린에서는 null은 매우 중요해요.
null이 왜 중요할까요? 그리고 null을 어떻게 해야피할 수 있을까요?

Reply:
우선 null 이 중요한 이유는 kotlin은 자바와 다르게 null을 처리할 수 있는 타입이 따로 제공됩니다. 그렇기 때문에 nullable 하지 않은 변수에 null이 들어오면 문제가 생기게 됩니다.
해결 방안은 nullable 타입으로 변경하거나 엘비스 연산자와 같이 안전하게 호출할 수 있도록 변경하면 될 것 같습니다.

Reply:
좋아요! 그거에요 : ) 저는 항상 null에 대해 안전하도록 코드를 짜고 있습니다 + ㅁ+